jitrebalance/test_jitrebalance.py

```diff
@@ -0,0 +1,79 @@
+from pyln.testing.fixtures import *  # noqa: F401, F403
+from pyln.testing.utils import wait_for
+from pprint import pprint
+import os
+import time
+
+
+plugin = os.path.join(os.path.dirname(__file__), 'jitrebalance.py')
+
+
+@unittest.skipIf(not DEVELOPER, "gossip is too slow if we're not in developer mode")
```

Comment:
As Travis points out, there is missing dependencies for this annotation:

https://travis-ci.org/lightningd/plugins/jobs/620908677
```
jitrebalance/test_jitrebalance.py:11: in <module>
    @unittest.skipIf(not DEVELOPER, "gossip is too slow if we're not in developer mode")
E   NameError: name 'unittest' is not defined
```